plugins/no-content/no-content.js

```diff
@@ -0,0 +1,36 @@
+const stylelint = require('stylelint');
+
+const ruleName = 'plugin/no-content';
+
+const messages = stylelint.utils.ruleMessages(ruleName, {
+  rejected: 'You must not use the `content` property to hard-code unlocalized text',
+});
+
+module.exports = stylelint.createPlugin(ruleName, (primaryOption) => {
```

Comment:
I was under the impression that we needed to create a custom rule for this, not a plugin. The tests suggest we're testing a rule, not a plugin.

Reply:
Blame stylelint for the language confusion. :P

In stylelint land, a [plugin](https://stylelint.io/user-guide/configuration/#plugins) _is_ a custom rule (or set of rules). This repo would be a [third-party config](https://stylelint.io/user-guide/configuration/#extends)

Reply:
Thanks for the clarification!